api/src/main/java/org/openmrs/module/appointments/model/AppointmentPriority.java

```diff
@@ -0,0 +1,13 @@
+package org.openmrs.module.appointments.model;
+
+public enum AppointmentPriority {
+    AsNeeded("AsNeeded"), Routine("Routine"), Emergency("Emergency"), Invalid("Invalid");
```

Comment:
@kavitha-sundararajan confirming these are the FHIR priority status ? I remember there is "stat" as status

Reply:
Yes @binduak , Akhil has gone through the different types and suggested use these three "AsNeeded, Routing and Emergency" and we added "Invalid" for identifying invalid types coming from the API

Reply:
ok..these are FHIR status ?

Reply:
Others except "Invalid" are FHIR status [https://terminology.hl7.org/5.1.0/ValueSet-v3-ActPriority.html]. Is it okay to have that ?

Reply:
yes @kavitha-sundararajan